_includes/setup/get-sdk-win.md

```diff
@@ -0,0 +1,59 @@
+## Get the Flutter SDK
+
+To get Flutter, use `git` to clone the repository and then add the `flutter` tool to your path.
+Running `flutter doctor` shows any remaining dependencies you may need to install.
+
+### Clone the repo
+
+If this is the first time you're installing Flutter on this machine, clone the
+repository and then add the `flutter` tool to your path:
+
+```
+> git clone https://github.com/flutter/flutter.git
+> setx path "%path%;%cd%\flutter\bin"
+```
```

Comment:
We should mention that you need to open a new terminal for this to take effect. Otherwise, `flutter doctor` in the next step will just fail.

From https://ss64.com/nt/setx.html:
> Because SETX writes variables to the master environment in the registry, edits will only take effect when a new command window is opened - they do not affect the current CMD or PowerShell session.